ReactCommon/react/renderer/components/view/primitives.h

```diff
@@ -160,12 +160,24 @@ struct CascadedRectangleCorners {
   OptionalT bottomStart{};
   OptionalT bottomEnd{};
   OptionalT all{};
+  OptionalT endEnd{};
+  OptionalT endStart{};
+  OptionalT startEnd{};
+  OptionalT startStart{};
 
   Counterpart resolve(bool isRTL, T defaults) const {
-    const auto topLeading = isRTL ? topEnd : topStart;
-    const auto topTrailing = isRTL ? topStart : topEnd;
-    const auto bottomLeading = isRTL ? bottomEnd : bottomStart;
-    const auto bottomTrailing = isRTL ? bottomStart : bottomEnd;
+    const auto topLeading = isRTL ? startEnd ? startEnd : topEnd
+        : startStart              ? startStart
+                                  : topStart;
```

Comment:
Same comment as elsewhere about precedence, but this ternary is a bit hard to read as-is.  Could we avoid the nested ternary (or maybe this is fine when formatted)?

Reply:
Yeah that logic was pretty confusing, I've just updated it making things a bit easier to read